packages/eslint-plugin/lib/use-jessie-rules.js

```diff
@@ -5,72 +5,62 @@
 const nono = `not allowed in Jessie`;
 
 exports.jessieRules = {
+  '@jessie.js/no-nested-await': ['error'],
   curly: ['error', 'all'],
   eqeqeq: ['error', 'always'],
   'no-bitwise': ['error'],
   'no-fallthrough': ['error', { commentPattern: `fallthrough is ${nono}` }],
-  'no-restricted-globals': ['error', 'RegExp', 'Date'],
+  // The denylist is probably too permissive.  We should have an allowlist.
+  'no-restricted-globals': ['error', 'RegExp', 'Date', 'Symbol'],
+  // Hint to future readers: https://eslint.org/docs/developer-guide/selectors
+  // is a guide to the `no-restricted-syntax` rule configuration.  You can use
+  // https://astexplorer.net/#/gist/4508eec25a8d5be1e0248c4cc06b9634/f6b22f2e8e3abd82a911ca6286a304ef0a3018c4
+  // to see what AST nodes are produced by different programs.
   'no-restricted-syntax': [
     'error',
     {
       selector: `BinaryExpression[operator='in']`,
       message: `'in' is ${nono}`,
     },
-    {
-      selector: `UpdateExpression[operator='++'][prefix=false]`,
-      message: `postfix '++' is ${nono}`,
-    },
-    {
-      selector: `UpdateExpression[operator='--'][prefix=false]`,
-      message: `postfix '--' is ${nono}`,
-    },
     {
       selector: `BinaryExpression[operator='instanceof']`,
-      message: `'instanceof' is ${nono}`,
+      message: `'instanceof' is ${nono}; use duck typing`,
     },
     {
       selector: `NewExpression`,
-      message: `'new' is ${nono}`,
+      message: `'new' is ${nono}; use a 'maker' function`,
     },
     {
       selector: `FunctionDeclaration[generator=true]`,
       message: `generators are ${nono}`,
     },
     {
-      selector: `FunctionDeclaration[async=true]`,
-      message: `async functions are ${nono}`,
-    },
-    {
-      selector: `FunctionExpression[async=true]`,
-      message: `async functions are ${nono}`,
-    },
-    {
-      selector: `ArrowFunctionExpression[async=true]`,
-      message: `async functions are ${nono}`,
+      selector: `FunctionExpression[generator=true]`,
+      message: `generators are ${nono}`,
```

Comment:
Do we have library function alternatives to generators?

Reply:
In progress.  It takes some explanation, though, so I'm not sure yet what to put in the linting failure.